build.cake

```diff
@@ -207,6 +213,7 @@ Task("Publish-NuGet")
     .WithCriteria(() => parameters.IsMainCakeRepo)
     .WithCriteria(() => parameters.IsMainCakeBranch)
     .WithCriteria(() => parameters.IsTagged)
+    .ContinueOnError()
```

Comment:
@gep13 I think the easiest way to test this is a new Cake script with both `ContinueOnError` and `OnError` where you throw an exception in the task.